calculator.rb

```diff
@@ -0,0 +1,151 @@
+#operation methods
+def add(num1, num2)
+  return num1 + num2
+end
+
+def subtract(num1, num2)
+  return num1 - num2
+end
+
+def multiply(num1, num2)
+  return num1 * num2
+end
+
+def divide(num1, num2)
+  return num1 / num2
+end
+
+def modulo(num1, num2)
+  return num1 % num2
+end
+
+def exponent(num1, num2)
+  return num1 ** num2
+end
+
+#request number methods
+def requestNum1
+  puts "We will be working with two numbers. Please enter your first number:"
+  num = gets.chomp
+  
+  return num
+end
+
+def requestNum2
+  puts "Please enter your second number:"
+  num = gets.chomp
+  
+  return num
+end
+
+valid = false
+OPERATIONS = ["add", "+", "subtract", "-", "multiply", "*", "divide", "/", "modulo", "%", "exponent", "**"]
+
+puts "Welcome to the Calculator program! Which operator would you like to use?"
+
+puts "1. add(+)"
+puts "2. subtract(-)"
+puts "3. multiply(*)"
+puts "4. divide(/)"
+puts "5. modulo(%)"
+puts "6. exponent(**)"
+
+
+
+while valid == false 
+  
+  puts "Please choose an operator(name or symbol):"
+  operator = gets.chomp
+  operator = operator.downcase
+  
+  if OPERATIONS.include? operator
```

Comment:
These nested `if` statements get really challenging to read the further they get nested.  How would you write this program without nesting `if`s so deeply?